internal/services/cdn/cdn_frontdoor_custom_domain_resource.go

```diff
@@ -55,15 +53,6 @@ func resourceCdnFrontDoorCustomDomain() *pluginsdk.Resource {
 				ValidateFunc: validate.FrontDoorProfileID,
 			},
 
-			// NOTE: I need the reference to ensure the correct destroy order
-			// this will also offload the task of maintaining the custom domain association
-			// IDs in the route resource to the custom domains themselves...
-			"associate_with_cdn_frontdoor_route_id": {
-				Type:         pluginsdk.TypeString,
-				Optional:     true,
-				ValidateFunc: validate.FrontDoorRouteID,
-			},
```

Comment:
this is a breaking change

Reply:
Yes it is but it has to be else you get the `cycle` error when you attempt to run terraform if both of these fields exist in the configuration file.

Reply:
as discussed internally msft is aware of this and ok with it